src/app/app.component.ts

```diff
@@ -1,8 +1,9 @@
-import { Component } from "@angular/core";
+import { Component } from '@angular/core';
 
 @Component({
-  selector: "app-root",
-  template: "<router-outlet></router-outlet>",
+  selector: 'app-root',
+  template: `<app-navbar [notifications]="99"></app-navbar
```

Comment:
czemu to leci do app component a nie do `home`? app sluzy tylko jako placeholder dla całej aplikacji

Reply:
Done